packages/core/src/item.ts

```diff
@@ -323,14 +323,16 @@ export class VaultItem extends Serializable {
     @AsDate()
     lastAudited?: Date = undefined;
 
-    expiresEveryPeriodOfDays?: number = undefined;
+    /** Number of days after which the item expires */
+    expiresAfter?: number = undefined;
```

Comment:
I know adding the unit to the property name can be considered good practice, but since we're not doing anywhere else (see https://github.com/padloc/padloc/blob/main/packages/core/src/app.ts#L49-L53 for example), I'd rather keep things consistent.